pandas/plotting/_core.py

```diff
@@ -1667,8 +956,123 @@ def hexbin(self, x, y, C=None, reduce_C_function=None, gridsize=None,
             ...                     gridsize=10,
             ...                     cmap="viridis")
         """
-        if reduce_C_function is not None:
-            kwds['reduce_C_function'] = reduce_C_function
-        if gridsize is not None:
-            kwds['gridsize'] = gridsize
-        return self(kind='hexbin', x=x, y=y, C=C, **kwds)
+        if not isinstance(self._parent,
+                          pandas.core.dtypes.generic.ABCDataFrame):
+            raise ValueError(
+                "plot kind hexbin can only be used for data frames")
+        return self(kind='hexbin', x=x, y=y, C=C,
+                    reduce_C_function=reduce_C_function, gridsize=gridsize,
+                    **kwargs)
+
+
+def hist_series(self, by=None, ax=None, grid=True, xlabelsize=None,
+                xrot=None, ylabelsize=None, yrot=None, figsize=None,
+                bins=10, **kwds):
+    plot_backend = _get_plot_backend()
+    return plot_backend.hist_series(self, by=by, ax=ax, grid=grid,
+                                    xlabelsize=xlabelsize, xrot=xrot,
+                                    ylabelsize=ylabelsize, yrot=yrot,
+                                    figsize=figsize, bins=bins, **kwds)
+
+
+hist_series.__doc__ = PlotAccessor.hist.__doc__
```

Comment:
I would leave the original hist_series and hist_frame intact, as the accessor's docstring version is not correct for them (different keywords, different order of the keywords, ...)

Reply:
And the same for boxplot below

Reply:
Didn't realize, good point, thanks!